test/common/stats/tag_extractor_impl_test.cc

```diff
@@ -156,9 +156,9 @@ TEST(TagExtractorTest, DefaultTagExtractors) {
   // Cluster name
   Tag cluster_tag;
   cluster_tag.name_ = tag_names.CLUSTER_NAME;
-  cluster_tag.value_ = "ratelimit";
+  cluster_tag.value_ = "rate_limit-14";
 
-  regex_tester.testRegex("cluster.ratelimit.upstream_rq_timeout", "cluster.upstream_rq_timeout",
+  regex_tester.testRegex("cluster.rate_limit-14.upstream_rq_timeout", "cluster.upstream_rq_timeout",
```

Comment:
nit: you could revert the changes to add dashes as dashes are no longer special in your regexes.